src/backend/app.js

```diff
@@ -4,32 +4,139 @@ const router = express.Router();
 const path = require("path");
 
 const mealsRouter = require("./api/meals");
+const reservationsRouter = require("./api/reservations");
+
 const buildPath = path.join(__dirname, "../../dist");
 const port = process.env.PORT || 3000;
 const cors = require("cors");
 
-// For week4 no need to look into this!
-// Serve the built client html
+//to get connected to the database
+const knex = require("./database");
+
 app.use(express.static(buildPath));
 
 // Parse URL-encoded bodies (as sent by HTML forms)
 app.use(express.urlencoded({ extended: true }));
+
 // Parse JSON bodies (as sent by API clients)
 app.use(express.json());
 
 app.use(cors());
 
 router.use("/meals", mealsRouter);
+router.use("/reservations", reservationsRouter);
+
+//________________________________________________
+
+//-----week1-----
+
+const todayDate = new Date();
+app.get("/future-meals", async (req, res) => {
```

Comment:
Here I think the endpoint address could look like this: `meals/future`  - it is using the same resource as previous meals endpoints